source/views/directory/detail.ios.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react'
+import {ScrollView, Text, StyleSheet, Image} from 'react-native'
+import {openUrl} from '@frogpond/open-url'
+import {callPhone} from '../../components/call-phone'
+import {sendEmail} from '../../components/send-email'
+import {Title, Detail} from '@frogpond/lists'
+import {
+	TableView,
+	Section,
+	Cell,
+	MultiLineLeftDetailCell,
+} from '@frogpond/tableview'
+import * as c from '@frogpond/colors'
+import type {DirectoryItem, Department, CampusLocation} from './types'
+import type {TopLevelViewPropsTypeWithParams} from '../types'
+
+type Props = TopLevelViewPropsTypeWithParams<{contact: DirectoryItem}>
+
+export function DirectoryDetailView(props: Props): JSX.Element {
+	const {
+		displayName,
+		campusLocations,
+		displayTitle,
+		photo,
+		officeHours,
+		profileUrl,
+		email,
+		departments,
+	} = props.navigation.state.params.contact
+
+	return (
+		<ScrollView>
+			<Image
+				accessibilityIgnoresInvertColors={true}
+				resizeMode="cover"
+				source={{uri: photo}}
+				style={styles.image}
+			/>
+			<Title style={[styles.header, styles.headerName]}>{displayName}</Title>
+			<Detail style={[styles.header, styles.headerTitle]}>
+				{displayTitle}
+			</Detail>
+
+			<TableView>
+				{officeHours || email || profileUrl ? (
+					<Section header="ABOUT">
+						{email ? (
+							<Cell
+								accessory="DisclosureIndicator"
+								cellStyle="LeftDetail"
+								detail="Email"
+								onPress={() => sendEmail({to: [email], subject: '', body: ''})}
+								title={email}
+							/>
+						) : null}
+
+						{officeHours && (
+							<MultiLineLeftDetailCell
+								accessory={officeHours.href ? 'DisclosureIndicator' : undefined}
+								detail={officeHours.title}
+								onPress={
+									officeHours.href ? () => openUrl(officeHours.href) : null
```

Comment:
What do we think of this solution? The type that ships for `CellInterface['onPress']` looks like an oversight as we know we are able to return, pass, and do things other than the provided signature `(() => any) | undefined)`.

I am suggesting a replacement `onPress` type because the one that ships with the library is too restrictive. I would prefer if we overwrite it and use our own less restrictive `onPress`.

```ts
type CellInterfaceModifiedType = Omit<CellInterface, 'onPress'> & {
	onPress?: () => void   // (or some variant of `void | undefined`)
}
```



Does this sound okay to you, @rye?